node/package.json

```diff
@@ -18,7 +17,7 @@
     "@types/node": "^7.0.8",
     "@types/ramda": "^0.26.8",
     "@types/route-parser": "^0.1.2",
-    "@vtex/api": "^3.24.1",
+    "@vtex/api": "^3.50.2",
     "tslint": "^5.11.0",
     "tslint-config-vtex": "^2.1.0",
     "typescript": "3.5.2"
```

Comment:
Missing changelog entry.

Reply:
This project don't have changelog, I will not add this for now.

Reply:
Any time is time to add a changelog imo but ok.